env/env.cc

```diff
@@ -735,6 +736,8 @@ Status Env::GetChildrenFileAttributes(const std::string& dir,
 
 Status Env::GetHostNameString(std::string* result) {
   std::array<char, kMaxHostNameLen> hostname_buf;
+  // make sure array contents are properly zeroed
+  std::fill(hostname_buf.begin(), hostname_buf.end(), '\0');
```

Comment:
A much more concise way to do this is put `{}` after `hostname_buf` in declaration.

Reply:
Right. Sorry I was a bit too verbose. Fixed it.